app/lambdas/notify_slack_py/notify_slack.py

```diff
@@ -0,0 +1,81 @@
+import json, urllib.request
+import boto3
+
+
+from orcabus_api_tools.data_sharing import get_data_sharing_url
+from orcabus_api_tools.utils.requests_helpers import get_request
+
+
+# Should haddle this better in the future, defining it in constants and so on...
+def _get_webhook_url():
+    _sm = boto3.client("secretsmanager")
+    return _sm.get_secret_value(SecretId="auto-data-sharing-slack-webhook")["SecretString"] # pragma: allowlist secret
+
+
+
+
+def get_package_report(package_id):
+
+    packaging_report_api_url = get_data_sharing_url(f"/api/v1/package/{package_id}:getSummaryReport")
+
+    return get_request(
+        url=packaging_report_api_url,
+    )
+
+
+def handler(event, context):
+    WEBHOOK_URL = _get_webhook_url()
+    id = event["id"]
```

Comment:
PyCharm will show a warning for using 'id' as a variable name in python. Can you rename to orcabus_id?